src/components/Calculator.jsx

```diff
@@ -1,27 +1,31 @@
-/* eslint-disable react/prop-types */
-import React from 'react';
-// import calculate from '../logic/calculate';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
 import Buttons from './Buttons';
 import Display from './Display';
 
-/* eslint-disable react/prefer-stateless-function */
-/* eslint-disable react/destructuring-assignment */
+const Calculator = () => {
+  const [state, setState] = useState({
+    total: '0',
+    next: null,
+    operation: null,
+  });
 
-class Calculator extends React.Component {
-  render() {
-    return (
-      <div className="calculator">
-        <Display
-          total={this.props.total}
-          operation={this.props.operation}
-          next={this.props.next}
-        />
-        <Buttons
-          clickHandler={this.props.clickHandler}
-        />
-      </div>
-    );
-  }
-}
+  const clickHandler = (event) => {
+    setState({ ...setState, ...calculate(state, event) });
+  };
+
+  return (
+    <div className="calculator">
+      <Display
```

Comment:
- [ ] `required`
- [ ] great job implementing the full site 👏 
- [ ] however there is need for adjusting in accordance to the template.
- [ ] please move the calculator to the `right` as well on left so as to match the wireframe. 🥂 
- [ ] The layout should match the wireframe please see screenshots below and marked areas.

`your template`
![image](https://user-images.githubusercontent.com/66230306/200111625-819e03e1-bb96-4344-b374-9dc075eb9070.png)

`original template`
![image](https://user-images.githubusercontent.com/66230306/200111651-6e6ebac9-976b-48f6-b3dc-348c9257c3f1.png)